.github/workflows/static.yml

```diff
@@ -1,43 +1,33 @@
 name: golang/static
 on: pull_request
-
 jobs:
-
   vet:
     name: vet
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: vet
-      uses: grandcolline/golang-github-actions@v1.1.0
-      with:
-        run: vet
-        comment: false
-  
-  # TODO: Enable linters below one by one and fix issues with each
-
+    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v2
+    - run: go mod download
+    - run: go vet ./core/...
   shadow:
-    name: Shadow
+    name: shadow
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-      - name: check
-        uses: grandcolline/golang-github-actions@v1.1.0
-        with:
-          run: shadow
-          comment: false
-
+    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v2
```

Comment:
Typically checkouts go first in a workflow

Reply:
I reordered as you suggested.